utils/licenses.js

```diff
@@ -58,6 +58,12 @@ export const licenses = [
 	},
 ];
 
-export const getLicenseBySlug = (slug = 'cc-by') => {
-	return licenses.find((ls) => ls.slug === slug);
+export const getLicenseBySlug = (slug = 'cc-by', publisherString, copyrightYear) => {
+	const license = licenses.find((ls) => ls.slug === slug);
+	license.summary = `${license.slug.toUpperCase()} ${license.version}`;
```

Comment:
This will modify the object in `licenses` for future callers. You'd want to shallow-copy it like this before mutating it:

```ts
const license = { ...licenses.find((ls) => ls.slug === slug) };
```